### PR TITLE
chore: create person card component (WEB-43)

### PR DESCRIPTION
**Pull request by Ayomide4**
Screenshot 2024-05-30 at 10 51 35 AM

Created person component with tests.

### CODE REVIEW

File: src/app/components/person/person.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  twitterUrl: string;
+  linkedinUrl: string;
+  imageUrl: string;
+}
+
+export default function Person({
+  fullName,
+  twitterUrl,
+  linkedinUrl,
+  imageUrl,
```
Comment (GetBlackBoxSolutions) [inline src/app/components/person/person.tsx:14]: Image url is unused. Did you mean to use this in "src" below?

Comment (Ayomide4) [reply src/app/components/person/person.tsx:14]: Yeah, should be fixed now!

**Review by GetBlackBoxSolutions — CHANGES_REQUESTED** [review]

File: src/app/components/person/person.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  fullName: string;
+  twitterUrl: string;
+  linkedinUrl: string;
+  imageUrl: string;
+}
+
+export default function Person({
+  fullName,
+  twitterUrl,
+  linkedinUrl,
```
[the remark this answers is not in the document]
Comment (Ayomide4) [reply src/app/components/person/person.tsx:14]: Screenshot 2024-06-24 at 2 24 32 PM
Screenshot 2024-06-24 at 2 25 47 PM
The first image is the latest commit when you don't enter fullName, imageUrl, twitterUrl and linkedinUrl
The second image is when you enter all except imageUrl

**Comment by Ayomide4** [comment]
Screenshot 2024-06-24 at 3 12 17 PM

ImageUrl looked off vs default image url. Fixed now.

**Review by GetBlackBoxSolutions — APPROVED** [review]

**Review by GetBlackBoxSolutions — APPROVED** [review]

**Pull request merged**